lib/langchain/conversation.rb

```diff
@@ -37,6 +37,11 @@ def initialize(llm:, **options, &block)
       @block = block
     end
 
+    def set_functions(functions)
+      @llm.functions = functions
+      @llm.complete_chunk = true
```

Comment:
Can we rename this variable to something else?

"chunk" refers to something very specific in the LLM world.

Perhaps "complete_response"?